scripts/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Template build script
+# ---------------------
+
+rm -rf CMakeCache.txt CMakeFiles
+
+export CXX=FIXME
+export MKLROOT=FIXME
+HPX_ROOT=FIXME
+BLASPP_ROOT=FIXME
+LAPACKPP_ROOT=FIXME
+
+# CMAKE_BUILD_TYPE := Release | RelWithDebugInfo | Debug
+#
+# DLAF_WITH_CUDA   := ON | OFF (default)
+#
+# DLAF_WITH_MKL    := ON | OFF (default)
+#
+# BUILD_DOC        := ON | OFF (default)
+#
+# DLAF_WITH_TEST   := ON (default) | OFF 
+#
+cmake ~/code/DLA-Future \
```

Comment:
I would not assume that source code will always be in `$HOME/code`.

I would replace this absolute assumption with a relative one. A relative assumption may be that the source folder is the parent one. This is not the only option, we can evaluate to assume that the build folder is next to the source folder and not nested in, but we will need to make another assumption about the name of the source folder.

I prefer the first option. What do you think?

Reply:
That was an error on my part, I didn't notice that I hadn't add FIXME for the path to the source directory. I don't think we should make any assumptions on where anyone might want to place the project. I will just leave it to the user to specify the path.